tap_github/client.py

```diff
@@ -266,10 +269,68 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
         resp_json = response.json()
         yield from extract_jsonpath(self.query_jsonpath, input=resp_json)
 
+    def get_next_page_token(
+        self, response: requests.Response, previous_token: Optional[Any]
+    ) -> Optional[Any]:
+        """
+        Return a dict of cursors for identifying next page or None if no more pages.
+
+        Note - pagination requires the Graphql query to have nextPageCursor_X parameters
+        with the assosciated hasNextPage_X, startCursor_X and endCursor_X.
+
+        X should be an integer between 0 and 9, increasing with query depth.
+
+        Warning - we recommend to avoid using deep pagination.
```

Comment:
@ericboucher Sorry, just catching up. Is the query depth controlled by `Stream.query`?

Reply:
Yes, basically how many `nextPageCursor_X` you would have and how `nested` they would be.

Reply:
I added `(nested)` to that comment.